### ISSUE TITLE
Hovering the mouse pointer over files with the right button of the mouse pressed should mark those files as selected

### ISSUE THREAD

**Issue by strangeattr** [MeanEYE/Sunflower#279]
As long as the mouse right button is held pressed and the mouse cursor is moved within the boundaries of the same panel, every file the cursor flies over should get marked as selected (the way it works in Midnight Commander).

**Comment by MeanEYE** [comment]
Good idea. I'll mark this as feature request.

**Comment by MeanEYE** [comment]
But the target is different, no?

**Comment by ArseniyK** [comment]
target can be directory in the same panel

**Comment by MeanEYE** [comment]
Ah right you are. Any proposition on how to solve this?

One of the solutions would certainly be to always show detailed menu on drag and drop so right mouse gets cleared. Or some similar behavior.

**Comment by ArseniyK** [comment]
Spied in MS Explorer:
if item already selected - init dnd else init selecting

**Comment by MeanEYE** [comment]
Might not be a bad idea. Although in such case DND of single item might require selecting it beforehand.

**Comment by ArseniyK** [comment]
works only on whitespace: so if rigth click directly on text in name column just start dnd

**Comment by MeanEYE** [comment]
Yeah but in such case if you want to select from the middle and go up you would start DND.
